src/js/main.js

```diff
@@ -1,6 +1,21 @@
-import {animatedScrollTo} from './es6-scroll-to.js';
+// using only ES6
 
-document.querySelector('.title').innerHTML = 'Great success!'
-document.querySelector('.text').innerHTML = 'Javascript is working!'
 
-animatedScrollTo(500);
\ No newline at end of file
+const buttons = document.querySelectorAll('button');
+buttons.forEach(btnClick => btnClick.addEventListener('click', openInfobox));
+
+function openInfobox(event) {
+    const btn = event.target;
+    const elementBlockSelector = 'contactUs';
```

Comment:
```suggestion
    const elementBlockSelector = '.contactUs';
```

Reply:
Then you should make 2 variables, one with the selector (including the dot or the hashtag respectively) and one with the classname only.

Reply:
would this be better or just more confusing ❓ 
 (.${elementBlockSelector})

Reply:
More confusing.

Reply:
The best approach, however, is to initally save the element to a variable and then use the variable. So:
```
const exampleElementSelector = '.example__element';
const exampleElementClass = 'example__element';
const exampleElement = document.querySelector(exampleElementSelector);
```

Also, while your approach of putting the class in one variable, then the individual modifier-suffix in another one and then putting them together with a +, is kind of nice, it's very practical. In projects, I just put the whole modifiers in variables. So like this:

```
const elementBlockOpenModifier = '.contactUs--opened';
```

And I think that's more practical.